backend/src/LibParser/FSharpToWrittenTypes.fs

```diff
@@ -338,6 +340,8 @@ module Expr =
     | SynExpr.Const(SynConst.Int32 n, _) -> WT.EInt(id, n)
     | SynExpr.Const(SynConst.Int64 n, _) -> WT.EInt(id, int64 n)
     | SynExpr.Const(SynConst.UInt64 n, _) -> WT.EInt(id, int64 n)
+    | SynExpr.Const(SynConst.SByte n, _) -> WT.EInt8(id, int8 n)
+    | SynExpr.Const(SynConst.Byte n, _) -> WT.EInt8(id, int8 n)
```

Comment:
Could you please clarify this further? Did you mean that we need to do something like the following:
``` 
| SynExpr.Const(SynConst.Byte n, _) -> WT.EUInt8(id, uint8 n)
```
 and add a new EUInt8, TUInt8... in WT, PT, RT, etc.
 if so, should we have built-in functions for both signed and unsigned int8?

Reply:
Yes, exactly. And yes separate functions.